maps/z2.dmm

```diff
@@ -41902,12 +41903,12 @@
 /area/afterlife/heaven)
 "bZV" = (
 /obj/tombstone{
+<<<<<<< HEAD
```

Comment:
u fucked up on merging with all this stuff

Reply:
🙀 will fix it when I re-run the linters later tonight